### PR TITLE
Update env for staging

### PR DESCRIPTION
**Pull request by dawidgil**

### CODE REVIEW

File: scripts/stg-env.js [hunk 1]
```diff
@@ -6,8 +6,8 @@ fs.readFile(envConfig, 'utf8', function (err, data) {
         return console.log(err);
     }
     const result = data
-        .replace('cacheServerUrl = \'https://identitycache-dev.energyweb.org/v1\'', 'cacheServerUrl = \'https://volta-identitycache.energyweb.org/v1\'')
-        .replace('natsServerUrl = \'https://identityevents-dev.energyweb.org/\'', 'natsServerUrl = \'https://volta-identityevents.energyweb.org/\'')
+        .replace('cacheServerUrl = \'https://identitycache-dev.energyweb.org/v1\'', 'cacheServerUrl = \'https://identitycache-staging.energyweb.org//v1\'')
```
Comment (jrhender) [inline scripts/stg-env.js:9]: Is that additional slash correct?
```suggestion
        .replace('cacheServerUrl = \'https://identitycache-dev.energyweb.org/v1\'', 'cacheServerUrl = \'https://identitycache-staging.energyweb.org/v1\'')
```

File: scripts/stg-env.js [hunk 2]
```diff
@@ -6,8 +6,8 @@ fs.readFile(envConfig, 'utf8', function (err, data) {
...
+        .replace('natsServerUrl = \'https://identityevents-dev.energyweb.org/\'', 'natsServerUrl = \'wss://identityevents-staging.energyweb.org\'')
```
Comment (jrhender) [inline scripts/stg-env.js:10]: Are you pretty sure this change to `wss` from `https` is correct?

**Review by jrhender — COMMENTED** [review]
A couple questions

File: scripts/stg-env.js (hunk 1 above)
Comment (dawidgil) [reply scripts/stg-env.js:9]: probably not. I will remove it.

File: scripts/stg-env.js (hunk 2 above)
Comment (dawidgil) [reply scripts/stg-env.js:10]: Nope, i used value the you provided. I can check it.

Comment (jrhender) [reply scripts/stg-env.js:10]: Ah haha, yea, probably. Idk, I think better to use `https` even though it's weird and then change all the config to `wss` later
```suggestion
        .replace('natsServerUrl = \'https://identityevents-dev.energyweb.org/\'', 'natsServerUrl = \'https://identityevents-staging.energyweb.org\'')
```

**Comment by dawidgil** [comment]
No longer relevant. We switched back to angular environments for now.

**Pull request closed without merging**